### PR TITLE
Change text surrounding form

### PR DESCRIPTION
**Pull request by littledan**
- Removes copies of certain licenses that come after the form
  (the relevant license is linked to from the contract text)
- Replace text which precedes the form with an actual explanation
  of what the form is for.

### CODE REVIEW

**Comment by littledan** [comment]
Updated the text around the form to add further clarifications.

File: contributor/index.html [hunk 1]
```diff
@@ -14,40 +14,14 @@
...
-        </pre>
+<p>This form is used to grant a license to intellectual property rights related to contributions to ECMAScript standards ECMA-262 and ECMA-402, developed in the Ecma TC39 Royalty-Free Task Group. It is required in the following cases:</p>
+<ul>
+  <li>To attend a TC39 meeting as an invited expert and participate in committee discussion</li>
+  <li>To contribute a normative change to an ECMA-262, ECMA-402 or a proposal repository under the tc39 organization</li>
+</ul>
+<p>This form does not grant anyone the right to contribute to TC39 work in either of those ways.</p>
```
Comment (bterlson) [inline contributor/index.html]: "either of those ways"? I'm confused at what this line is saying.

**Review by bterlson — CHANGES_REQUESTED** [review]

File: contributor/index.html (hunk 1 above)
Comment (littledan) [reply contributor/index.html]: I meant, signing the form doesn't mean you have the right to go to a meeting, or mean that your PR will land. Maybe the clarification is especially important about meeting attendance, as it's kind of obvious that there is code review.

Comment (littledan) [reply contributor/index.html]: What do you think of the new text there?

**Comment by littledan** [comment]
OK, I've addressed the issue Brian mentioned, and he also remarked offline that the changes seem good. Sent to others and have not heard concerns. Landing the patch now; we can iterate on the clarifications but I think this is the main change needed.

**Pull request merged**